react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react"
+
+const content = [
+  {
+    lang: "HTML",
+    text: "Hypertext Markup Language (HTML) is the standard markup language for creating web pages and web applications. With Cascading Style Sheets (CSS) and JavaScript, it forms a triad of cornerstone technologies for the World Wide Web."
+  },
+  {
+    lang: "CSS",
+    text: "Cascading Style Sheets (CSS) is a style sheet language used for describing the presentation of a document written in a markup language such as HTML.CSS is a cornerstone technology of the World Wide Web, alongside HTML and JavaScript."
+  },
+  {
+    lang: "JS",
+    text: "JavaScript, often abbreviated as JS, is a high-level, interpreted programming language that conforms to the ECMAScript specification. JavaScript has curly-bracket syntax, dynamic typing, prototype-based object-orientation, and first-class functions."
+  }
+]
```

Comment:
You should add an `id` property to each of your items, this will help you know which item is open or closed

```JSX
const content = [
  {
    id: 1,
    lang: "HTML",
    text: "Hypertext Markup Language (HTML) is the standard markup language for creating web pages and web applications. With Cascading Style Sheets (CSS) and JavaScript, it forms a triad of cornerstone technologies for the World Wide Web."
  },
  {
    id: 2,
    lang: "CSS",
    text: "Cascading Style Sheets (CSS) is a style sheet language used for describing the presentation of a document written in a markup language such as HTML.CSS is a cornerstone technology of the World Wide Web, alongside HTML and JavaScript."
  },
  {
    id: 3,
    lang: "JS",
    text: "JavaScript, often abbreviated as JS, is a high-level, interpreted programming language that conforms to the ECMAScript specification. JavaScript has curly-bracket syntax, dynamic typing, prototype-based object-orientation, and first-class functions."
  }
]